__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Good stuff. Some of the other students used other methods to extract feature importance too. You should add those in!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>I certainly need to learn more methods for this purpose.</p><p>Also, I have only used my knowledge based on Chapter 1(ML). Perhaps, if I did use some of the methods from Chapter 2(DS), I could implement better methods and visualization to the project.</p>